.github/workflows/on_pull_request.yaml

```diff
@@ -11,3 +11,8 @@ jobs:
     permissions:
       pull-requests: read
     secrets: inherit
+    with:
+      rockcraft-channel: latest/edge
```

Comment:
same with on_push

Reply:
updated to stable.  I'd pin to a version, but they don't have versioned channels :(